Description: Fix a broken menu/node structure
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: vera/vera.texi
===================================================================
--- vera.orig/vera.texi	2021-01-05 13:31:57.345554122 -0500
+++ vera/vera.texi	2021-01-05 13:31:57.341554478 -0500
@@ -184,7 +184,7 @@
 @end ifinfo
 
 
-@node 0, A, Top, Top
+@node 0, A, Top,
 @c node-name, next, previous, up
 @unnumbered --- 0 ---
 @cindex 0, Acronyms beginning with
@@ -193,7 +193,7 @@
 @include vera.0
 
 
-@node A, B, 0, Top
+@node A, B, 0,
 @c node-name, next, previous, up
 @unnumbered --- A ---
 @cindex A, Acronyms beginning with
@@ -201,7 +201,7 @@
 @include vera.a
 
 
-@node B, C, A, Top
+@node B, C, A,
 @c node-name, next, previous, up
 @unnumbered --- B ---
 @cindex B, Acronyms beginning with
@@ -209,7 +209,7 @@
 @include vera.b
 
 
-@node C, D, B, Top
+@node C, D, B,
 @c node-name, next, previous, up
 @unnumbered --- C ---
 @cindex C, Acronyms beginning with
@@ -217,7 +217,7 @@
 @include vera.c
 
 
-@node D, E, C, Top
+@node D, E, C,
 @c node-name, next, previous, up
 @unnumbered --- D ---
 @cindex D, Acronyms beginning with
@@ -225,7 +225,7 @@
 @include vera.d
 
 
-@node E, F, D, Top
+@node E, F, D,
 @c node-name, next, previous, up
 @unnumbered --- E ---
 @cindex E, Acronyms beginning with
@@ -233,7 +233,7 @@
 @include vera.e
 
 
-@node F, G, E, Top
+@node F, G, E,
 @c node-name, next, previous, up
 @unnumbered --- F ---
 @cindex F, Acronyms beginning with
@@ -241,7 +241,7 @@
 @include vera.f
 
 
-@node G, H, F, Top
+@node G, H, F,
 @c node-name, next, previous, up
 @unnumbered --- G ---
 @cindex G, Acronyms beginning with
@@ -249,7 +249,7 @@
 @include vera.g
 
 
-@node H, I, G, Top
+@node H, I, G,
 @c node-name, next, previous, up
 @unnumbered --- H ---
 @cindex H, Acronyms beginning with
@@ -257,7 +257,7 @@
 @include vera.h
 
 
-@node I, J, H, Top
+@node I, J, H,
 @c node-name, next, previous, up
 @unnumbered --- I ---
 @cindex I, Acronyms beginning with
@@ -265,7 +265,7 @@
 @include vera.i
 
 
-@node J, K, I, Top
+@node J, K, I,
 @c node-name, next, previous, up
 @unnumbered --- J ---
 @cindex J, Acronyms beginning with
@@ -273,7 +273,7 @@
 @include vera.j
 
 
-@node K, L, J, Top
+@node K, L, J,
 @c node-name, next, previous, up
 @unnumbered --- K ---
 @cindex K, Acronyms beginning with
@@ -281,7 +281,7 @@
 @include vera.k
 
 
-@node L, M, K, Top
+@node L, M, K,
 @c node-name, next, previous, up
 @unnumbered --- L ---
 @cindex L, Acronyms beginning with
@@ -289,7 +289,7 @@
 @include vera.l
 
 
-@node M, N, L, Top
+@node M, N, L,
 @c node-name, next, previous, up
 @unnumbered --- M ---
 @cindex M, Acronyms beginning with
@@ -297,7 +297,7 @@
 @include vera.m
 
 
-@node N, O, M, Top
+@node N, O, M,
 @c node-name, next, previous, up
 @unnumbered --- N ---
 @cindex N, Acronyms beginning with
@@ -305,7 +305,7 @@
 @include vera.n
 
 
-@node O, P, N, Top
+@node O, P, N,
 @c node-name, next, previous, up
 @unnumbered --- O ---
 @cindex O, Acronyms beginning with
@@ -313,7 +313,7 @@
 @include vera.o
 
 
-@node P, Q, O, Top
+@node P, Q, O,
 @c node-name, next, previous, up
 @unnumbered --- P ---
 @cindex P, Acronyms beginning with
@@ -321,7 +321,7 @@
 @include vera.p
 
 
-@node Q, R, P, Top
+@node Q, R, P,
 @c node-name, next, previous, up
 @unnumbered --- Q ---
 @cindex Q, Acronyms beginning with
@@ -329,7 +329,7 @@
 @include vera.q
 
 
-@node R, S, Q, Top
+@node R, S, Q,
 @c node-name, next, previous, up
 @unnumbered --- R ---
 @cindex R, Acronyms beginning with
@@ -337,7 +337,7 @@
 @include vera.r
 
 
-@node S, T, R, Top
+@node S, T, R,
 @c node-name, next, previous, up
 @unnumbered --- S ---
 @cindex S, Acronyms beginning with
@@ -345,7 +345,7 @@
 @include vera.s
 
 
-@node T, U, S, Top
+@node T, U, S,
 @c node-name, next, previous, up
 @unnumbered --- T ---
 @cindex T, Acronyms beginning with
@@ -353,7 +353,7 @@
 @include vera.t
 
 
-@node U, V, T, Top
+@node U, V, T,
 @c node-name, next, previous, up
 @unnumbered --- U ---
 @cindex U, Acronyms beginning with
@@ -361,7 +361,7 @@
 @include vera.u
 
 
-@node V, W, U, Top
+@node V, W, U,
 @c node-name, next, previous, up
 @unnumbered --- V ---
 @cindex V, Acronyms beginning with
@@ -369,7 +369,7 @@
 @include vera.v
 
 
-@node W, X, V, Top
+@node W, X, V,
 @c node-name, next, previous, up
 @unnumbered --- W ---
 @cindex W, Acronyms beginning with
@@ -377,7 +377,7 @@
 @include vera.w
 
 
-@node X, Y, W, Top
+@node X, Y, W,
 @c node-name, next, previous, up
 @unnumbered --- X ---
 @cindex X, Acronyms beginning with
@@ -385,7 +385,7 @@
 @include vera.x
 
 
-@node Y, Z, X, Top
+@node Y, Z, X,
 @c node-name, next, previous, up
 @unnumbered --- Y ---
 @cindex Y, Acronyms beginning with
@@ -393,7 +393,7 @@
 @include vera.y
 
 
-@node Z, About , Y, Top
+@node Z, About , Y,
 @c node-name, next, previous, up
 @unnumbered --- Z ---
 @cindex Z, Acronyms beginning with
